packages/components/src/core/Autocomplete/index.tsx

```diff
@@ -101,14 +101,30 @@ const Autocomplete = <
     noOptionsText = "No options",
     onInputChange = noop,
     PaperComponent = StyledPaper,
-    PopperComponent = defaultPopperComponent,
```

Comment:
@tihuan, thank you for reviewing the code 🎉🙇🏻‍♂️ 
Since we are passing `{...props}` to the `<Autocomplete />` component later on, users can still define and use their own Popper with the SDS Autocomplete. 
I tested it and it works fine 🥸

Reply:
Ahh that's a really good point lol Thanks for letting me know and testing it!